### PR TITLE
Coverage Analysis - Add Refinements

### PR DESCRIPTION
**Pull request by silabs-robin**
This PR adds (some) manual refinements for code coverage.

It is the successor of https://github.com/openhwgroup/core-v-verif/pull/2150.

### CODE REVIEW

File: cv32e40s/sim/tools/xrun/README.md [hunk 1]
```diff
@@ -1,24 +1,28 @@
...
-| File | Maintenance | Description |
-|------|-------------|-------------|
-| cv32e40s.hierarchy.vRefine | Manual | Removes hierarchies from coverage database that are not to be considered for coverage (e.g. testbench |
-| cv32e40s.auto.vRefine | Automatic | Auto-generated refinements based on parameter usage for the CV32E40S without PULP extensions.  *Do not manually edit* |
-| cv32e40s.manual.vRefine | Manual | Manually added coverage exception based on deesign verification reviews. |
+| File                              | Maintenance | Description |
+|-----------------------------------|-------------|-------------|
+| cv32e40s.non_dut_code_cov.vRefine | Manual      | Removes non-DUT code coverage from coverage database, that are not to be considered for coverage (e.g. testbench) |
+| cv32e40s.auto.vRefine             | Automatic   | Auto-generated refinements based on parameter usage for the CV32E40S without PULP extensions.  *Do not manually edit* |
+| cv32e40s.manual.vRefine           | Manual      | Manually added coverage exception based on deesign verification reviews. |
```
Comment (kristinedosvik) [inline cv32e40s/sim/tools/xrun/README.md:28]: I might misunderstand this but is this cv32e40s.manual.vRefine a prefix of sufix? Or should it be cv32e40s.manual/non_dut_code_cov*.vRefine, as the files seems to be named so (f.eks. cv32e40s/sim/tools/xrun/cv32e40s.manual_block_cov_todo.vRefine and cv32e40s/sim/tools/xrun/cv32e40s.non_dut_code_cov_interfaces.vRefine. I guess it is not that important as I guess we understand the meaning considering that there is so many examples.

Comment (silabs-krdosvik) [inline cv32e40s/sim/tools/xrun/README.md:28]: I might misunderstand this but is this cv32e40s.manual.vRefine a prefix of sufix? Or should it be cv32e40s.manual/non_dut_code_cov*.vRefine, as the files seems to be named so (f.eks. cv32e40s/sim/tools/xrun/cv32e40s.manual_block_cov_todo.vRefine and cv32e40s/sim/tools/xrun/cv32e40s.non_dut_code_cov_interfaces.vRefine. I guess it is not that important as I guess we understand the meaning considering that there is so many examples.

**Comment by silabs-hfegran** [comment]
I did not see any unexpected waivers here, looks good

File: cv32e40s/sim/tools/xrun/README.md (hunk 1 above)
Comment (silabs-robin) [reply cv32e40s/sim/tools/xrun/README.md]: I've added asterisks now:
bilde

**Comment by silabs-robin** [comment]
Converting to draft.

New plan:
All coverage analysis to be done before PRing.

**Comment by silabs-robin** [comment]
Closing. New system.

**Pull request closed without merging**